### ISSUE TITLE
mdBottomSheet regression - grid button text not showing

### ISSUE THREAD

**Issue by Phr33d0m** [angular/material#7883]
Hello, as you can see in the [latest demo](https://material.angularjs.org/latest/demo/bottomSheet) when you click on the Grid example there are 2 issues:

1. The icons are huge
2. The button text is not showing as in previous versions <sup>[1]</sup>

![angular material 20 20 demos 20 20 bottom sheet](https://cloud.githubusercontent.com/assets/1591486/14279266/a141d0f6-fb2c-11e5-99ce-eafa41be7c02.png)

[1] As you can see in version 1.0.7 [it works as expected](https://material.angularjs.org/1.0.7/demo/bottomSheet)

Regards.

**Comment by RolandHeath** [comment]
If you look closely at that screenshot, it appears that the icons are actually cut off at the bottom as well. 
Looking in the devtools, you can confirm this: The text is there, but each button element is cut off at the wrong height, so you can't see it.

But definitely a confirmed issue in 1.1.0-rc2, with a different incorrect behaviour on HEAD.

**Comment by Phr33d0m** [comment]
> On HEAD, the text shows, and seems to align to the bottom of the icon button.

On 1.1.0-rc2 even if I forcefully make the icon smaller (as I already had that in my design) the text still does not appear.

Check this out:

![text-not-showing-mdbutton](https://cloud.githubusercontent.com/assets/1591486/14381297/bbcfa19a-fd85-11e5-83b7-ef9f20f1f33c.png)


Thanks to your investigation @RolandHeath I was able to do a workaround for my application<sup>[1]</sup>. If I click on the ```div._md-secondary-container``` (at the bottom of the screenshot) and hit DELETE the button text automagically appears.

On a side note I had to rewrite some of my mdLists because this new(?) _md-secondary-container bugged my design.


[1] CSS:
```css
md-list-item > div.flex + div._md-secondary-container {
    display: none;
}
```

Regards.
